com.unity.cinemachine/Samples~/Input System/Split Screen Multiplayer/SimpleControls.inputactions

```diff
@@ -0,0 +1,241 @@
+{
+    "name": "SimpleControls",
```

Comment:
These controls need tuning.  For example, joystick camera movement is way slower than mouse camera movement.  I suggest tuning both by adding processors to the controls, and then get rid of Gain in the input receiver, which is redundant.

Reply:
I can make it work for your mouse but I can't make something that will give the same result with every mouse. The reason is you might have a mouse with 800 dpi and an other one with 5000 dpi. 
I could add a processor to replace the gain though.